client/header/index.js

```diff
@@ -70,6 +70,7 @@ const getPageTitle = ( defaultTitle ) => {
 			payments: __( 'Set up payments', 'woocommerce-admin' ),
 			tax: __( 'Add tax rates', 'woocommerce-admin' ),
 			appearance: __( 'Personalize your store', 'woocommerce-admin' ),
+			marketing: __( 'Set up marketing tools', 'woocommerce-admin' ),
```

Comment:
I didn't even realize we were adding all task titles here.  This definitely makes me think we should be using `SlotFill` or some other means to encapsulate logic in their respective components.

This might be worth outlining an issue if you have time with the "cooldown" label so we can tackle this later.

Reply:
This definitely seems sub-par (existing issue, not a comment on this particular PR!) and should be addressed in the future!